test/test_collection.py

```diff
@@ -1849,6 +1849,8 @@ def test_exhaust(self):
         self.db.test.insert_many([{'i': i} for i in range(150)])
 
         client = rs_or_single_client(maxPoolSize=1)
+        self.addCleanup(client.close)
+        pool = get_pool(client)
         socks = get_pool(client).sockets
```

Comment:
We can probably remove this now and just use `pool.sockets` everywhere?

Reply:
Done.